spirv_reflect.h

```diff
@@ -262,6 +262,22 @@ typedef enum SpvReflectShaderStageFlagBits {
 
 } SpvReflectShaderStageFlagBits;
 
+/*! @enum SpvReflectAccessBits
+
+NOTE: A variable may be "accessed" but still have SPV_REFLECT_ACCESS_NONE
+      Example is if there is a OpAccessChain, but then it is never used
+
+*/
+typedef enum SpvReflectAccessFlagBits {
+  SPV_REFLECT_ACCESS_NONE   = 0x00000000,
+  SPV_REFLECT_ACCESS_READ   = 0x00000001,
+  SPV_REFLECT_ACCESS_WRITE  = 0x00000002,
+  // Atomic will always also be marked as READ and WRITE
+  SPV_REFLECT_ACCESS_ATOMIC = 0x00000004,
```

Comment:
Is there ever a case where ATOMIC is used or checked by itself it it always inlcudes READ and WRITE? Wondering if there's a different way to express this.

Reply:
so I had it before as separate... https://github.com/KhronosGroup/SPIRV-Reflect/issues/222#issuecomment-1724754338 convinced me otherwise